### PR TITLE
Rework: Disk sprites, Energy Guns, & New Terra Guns

### PR DESCRIPTION
**Pull request by Stutternov**
## About The Pull Request
<details>
<summary>
	About The Pull Request
</summary>
<hr>

Tl;dr - Redid nearly all energy weapon names, axed some redundant ones, and some other things. New disc icons, fixed serial numbers, and some other misc things.
	
<hr>
</details>

## Changelog
:cl:
removes: Redunant weapons and energy guns. Gun-bloat begone!!!
adds: New Terra compressed matter weapons.
adds: New disk sprites for all the factions; these should look more unique than before and some are antimated(!)
fixes: Few errors with gun serial numbers or manufacturers prior.
/:cl:

### CODE REVIEW

File: code/modules/projectiles/guns/projectile/matter_guns/matter_pistol.dm [hunk 1]
```diff
@@ -0,0 +1,91 @@
+/*
+Quirky Terra Therma weapon as requested by the department. Insert compressed matter from the Forge into this to fire it.
+No more 1 shot guns but hey, at least you no longer need cells and can carry a fat-stack of compressed matter instead.
+*/
+
+/obj/item/gun/projectile/matter_gun
+	name = "\"Forger\" compressed-matter pistol"
+	desc = "An odd design where a compressed matter cartriage is loaded into the firearm's designated port, effectively producing bullet-like particals."
```
Comment (WatermelonsEverywhere) [inline code/modules/projectiles/guns/projectile/matter_guns/matter_pistol.dm:8]: particals -> particles

File: code/modules/projectiles/guns/projectile/matter_guns/matter_pistol.dm [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+	return new projectile_type(src)
+
+/obj/item/gun/projectile/matter_gun/examine(user)
+	. = ..()
+	to_chat(user, "It holds [stored_matter]/[max_stored_matter] [matter_type].")
+
+
+/obj/item/gun/projectile/matter_gun/shotgun
+	name = "\"Forger\" compressed-matter pistol"
+	desc = "An odd design where a compressed matter cartriage is loaded into the firearm's designated port, effectively producing bullet-like particals. \
```
Comment (WatermelonsEverywhere) [inline code/modules/projectiles/guns/projectile/matter_guns/matter_pistol.dm:64]: particals -> particles here too, also the description makes it seem like a different gun based on the pistol above, but it has the same sprite path and name, so you only know it's one or the other by examining individually, is this intended?

Comment (Stutternov) [reply code/modules/projectiles/guns/projectile/matter_guns/matter_pistol.dm]: Amended & fixed. Was waiting on sprite improvements at the time; they never came. Set the correct DMI for their current uses. Spelling-errors also amended.

File: code/modules/projectiles/guns/projectile/matter_guns/matter_pistol.dm [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+	to_chat(user, "It holds [stored_matter]/[max_stored_matter] [matter_type].")
+
+
+/obj/item/gun/projectile/matter_gun/shotgun
+	name = "Mk II \"Forger\" compressed-matter pistol"
+	desc = "An odd design where a compressed matter cartriage is loaded into the firearm's designated port, effectively producing bullet-like particles. \
+			This version is far more bulkier than its pistol counterpart, but packs quite a punch. Able to fire projectiles that fragment on impact or incendiary rounds."
+	icon = 'icons/obj/guns/nano/shotgun.dmi'
+	icon_state = "railgun"
+	item_state = "railngun"
```
Comment (WatermelonsEverywhere) [inline code/modules/projectiles/guns/projectile/matter_guns/matter_pistol.dm:68]: Might be another typo  in the item_state here but I don't know the DMI for sure

Comment (Stutternov) [reply code/modules/projectiles/guns/projectile/matter_guns/matter_pistol.dm:68]: Fixed

**Pull request merged**